Add instructions to run analyses with npx (#33)
SonarSource/sonar-scanner-npm: README.md

npm install -g sonarqube-scanner
```

> To run analyses without installing the scanner at all, run the following command instead:

Reviewer: ```suggestion
To run analyses without explicitly installing the scanner, run the following command instead:
```

Reviewer: I'm slightly uncomfortable having this under `Installation`, would it fit better under `Usage` or somewhere else?

    `sonar-scanner -Dsonar.host.url=https://myserver.com -Dsonar.login=019d1e2e04e`

## Usage: run analyses with npx

To run analyses without explicitly installing the scanner, run the following command instead:

``` sh
npx sonarqube-scanner
```

> Similar to above, you can specify analysis properties and settings using either a `package.json` file, a `sonar-project.properties` file, or command line arguments.

Reviewer: ```suggestion
Similar to the above, you can specify analysis properties and settings using either a `package.json` file, a `sonar-project.properties` file, or command line arguments.
```
